lib/rubocop/cop/factory_bot/id_sequence.rb

```diff
@@ -24,7 +24,8 @@ class IdSequence < ::RuboCop::Cop::Base
         RESTRICT_ON_SEND = %i[sequence].freeze
 
         def on_send(node)
-          return unless node.first_argument.value == :id
+          return unless node.first_argument&.sym_type? &&
```

Comment:
WDYT of adding the check for the receiver (being FactoryBot or nil respectively) as suggested by @pirj ?

Reply:
Receiver checks were also added. What do you think of this?

Reply:
I think the check should be `factory_bot?(node.receiver)`
Also I'd place it first
```ruby
return unless factory_bot?(node.receiver)
return unless node.first_argument&.sym_type? && node.first_argument.value == :id
```

Reply:
Indeed it is. I updated this PR. Added a conditional expression since the receiver could be nil.